src/js/utils/MesosStateUtil.js

```diff
@@ -54,6 +54,41 @@ const MesosStateUtil = {
       });
   },
 
+  /**
+   * @param {Array.<Object>} resourceList - Verbose resource information
+   * @returns {Object} An object of only the resource values
+   */
+  extractExecutorResources(resourceList) {
```

Comment:
not entirely sure, but this should live in one of the mesosStream parsers I think 🤔- @nLight ?

Reply:
I took a look, but I don't see a util being added there.